src/composables/multisigAccounts.ts

```diff
@@ -175,12 +176,109 @@ export function useMultisigAccounts({
     }
   }
 
+  function splitArrayIntoFives(arr: any) {
+    const result = [];
+    for (let i = 0; i < arr.length; i += 5) {
+      result.push(arr.slice(i, i + 5));
+    }
+    return result;
+  }
+
   /**
-   * Refresh the list of the multisig accounts.
+   * Get extended data for a multisig account
    */
-  async function updateMultisigAccounts() {
+  async function getMultisigAccountInfo(
+    contractId: Encoded.ContractAddress,
+    gaAccountId: string,
+    otherMultisigData: any,
+  ): Promise<IMultisigAccount> {
     const dryAeSdk = await getDryAeSdk();
+    try {
+      const contractInstance = await dryAeSdk.initializeContract({
+        aci: SimpleGAMultiSigAci,
+        address: contractId,
+      });
```

Comment:
Maybe we can cache contract instances to avoid initializing them over and over?

Reply:
We can make it a separate task.